### PR TITLE
Change Sitemap 1000+ Warning to AIOSEOP_Notices

### PR DESCRIPTION
**Pull request by EkoJR**
Issue #1851

### CODE REVIEW

**Comment by arnaudbroes** [comment]
This is good, but the wording needs some tweaking. 

I'd recommend using something like this:

You've been using All in One SEO Pack for a while now. That's awesome! By giving us a 5-star review, you help us reach out to others.

Add a review.
Remind me later.
No, thanks.

What are your thoughts @michaeltorbert @wpsmort?

**Comment by arnaudbroes** [comment]
image
This looks good, although we might want to discuss two things @wpsmort @michaeltorbert:

1. All of the other notices have text action links. Do we want to change the buttons here to text links (for the sake of uniformity) or leave it as is?
2. More importantly, this notice currently only appears when Sitemap Indexes are already enabled. We also want to display this when Sitemap Indexes are disabled, but the total amount of URLs is higher than 1,000, right?

**Comment by michaeltorbert** [comment]
@arnaudbroes 

1. Is there a screenshot of what that looks like?
2. I could go either way on this, but if we go that route maybe we need to modify the message depending on whether they need to activate indexes or already have.

**Comment by wpsmort** [comment]
@arnaudbroes I prefer a button to a link

I would say that, if the action is part of the text of the message then it would be a link, but if it comes below the text message then it would be a button.

With regards to whether it should appear when Sitemap Indexes is disabled, we're enabling Sitemap Indexes by default (see issue #1929).  That means that the text will need to be changed.

**Comment by arnaudbroes** [comment]
@EkoJR change it to this copy and then this should be good to go:
"Notice: To avoid problems with your XML sitemap, we strongly recommend you set the Maximum Posts per Sitemap Page to 1,000."

**Comment by arnaudbroes** [comment]
@EkoJR I'd suggest changing the buttons to "XML Sitemap" and "Remind me later" just to make them uniform. Other than that, this looks great.

File: admin/functions-notice.php [hunk 1]
```diff
@@ -85,4 +85,78 @@ function aioseop_notice_disable_woocommerce_detected_on_nonpro() {
...
+				),
+
+			),
+		);
+	}
+
+	/**
+	 * Disable Notice for Sitemap
+	 *
+	 * @since 2.4.2
```
[the remark this answers is not in the document]
Comment (EkoJR) [reply admin/functions-notice.php]: Changed

**Pull request closed without merging**